certified-connectors/Baron Blue/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1492 @@
+                  "x-ms-visibility": "internal",
+                  "default": "raw"
+                },
+                "convert": {
+                  "type": "string",
+                  "description": "Please insert the file content to convert",
+                  "title": "CSV to convert"
+                },
+                "convert_to": {
+                  "type": "string",
+                  "description": "The type of file you want to get back",
+                  "title": "What do you want to convert it to?",
+                  "enum": [
+                    "JSON",
+                    "Excel 97-2004 (.xls)",
+                    "Excel (.xlsx)",
+                    "XML",
+                    "YAML"
+                  ]
+                }
+              },
+              "required": [
+                "convert_to",
+                "convert",
+                "return_type"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/v1/convert/file/xml/": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "converted": {
+                  "type": "string",
+                  "description": "The converted file from an XML file",
+                  "title": "Converted File"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "detail": {
+                  "type": "string",
+                  "description": "There was a problem with your request. Check the error message for more details.",
+                  "title": "Bad Request"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "detail": {
+                  "type": "string",
+                  "description": "There was a problem with your authentication. Try refreshing the connection or check if your subscription is still valid.",
+                  "title": "Unauthorized"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "detail": {
+                  "type": "string",
+                  "description": "There was an unexpected error. The development team was notified.",
+                  "title": "Unexpected error"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Convert a XML File",
+        "description": "Converts an XML file to the supported (XLS, XLSX, CSV, JSON, YAML)  file type",
```

Comment:
@mtgomes please make sure that descriptions are grammatically correct and are free of typographical errors. There's an extra whitespace in this string.